Description: make_converters.py gets pixel formats from color.h
 Necessary, because enum ALLEGRO_PIXEL_FORMAT was moved 
 from bitmap.h to color.h.
Author: Tobias Hansen <thansen@debian.org>
Applied-Upstream: http://sourceforge.net/p/alleg/allegro/ci/813cda5b5e6ef9d03034dea6bd84f103828ffb4b/

--- a/misc/make_converters.py
+++ b/misc/make_converters.py
@@ -4,7 +4,7 @@
 formats_by_name = {}
 formats_list = []
 
-def read_bitmap_h(filename):
+def read_color_h(filename):
     """
     Read in the list of formats.
     """
@@ -394,8 +394,8 @@
 functions."""
     options, args = p.parse_args()
 
-    # Read in bitmap.h to get the available formats.
-    formats = read_bitmap_h("include/allegro5/bitmap.h")
+    # Read in color.h to get the available formats.
+    formats = read_color_h("include/allegro5/color.h")
     
     print(formats)
 
